spring-cloud-common-security-config-web/src/main/java/org/springframework/cloud/common/security/support/ExternalOauth2ResourceAuthoritiesExtractor.java

```diff
@@ -44,7 +41,7 @@
  *
  * @author Mike Heath
  */
-public class ExternalOauth2ResourceAuthoritiesExtractor implements AuthoritiesExtractor {
+public class ExternalOauth2ResourceAuthoritiesExtractor implements AuthorityMapper {
```

Comment:
We can probably rename this class as this is now the `AuthorityMapper`.

Reply:
Fixed